packages/now-cli/src/util/dev/types.ts

```diff
@@ -100,7 +100,7 @@ export interface BuilderConfigAttr {
 }
 
 export interface Builder {
-  version?: 2;
+  version?: 2 | 3;
```

Comment:
I think you also need to update line 119 to be `output: BuilderOutputs | BuilderOutput;`

Reply:
@styfle probably, but we actually require the type in `119`, when the builder has version 3 we'll just transform it to be like this.

Reply:
@AndyBitz Let's not use type casting because its not safe. We can define the types correctly and then we won't need to cast.